mne/io/brainvision/brainvision.py

```diff
@@ -478,8 +483,10 @@ def _check_hdr_version(header):
 def _check_mrk_version(header):
     """Check the marker version."""
     tags = ['Brain Vision Data Exchange Marker File, Version 1.0',
-            'Brain Vision Data Exchange Marker File Version 1.0',
-            'Brain Vision Data Exchange Marker File, Version 2.0']
+            'BrainVision Data Exchange Marker File, Version 1.0',
+            'BrainVision Data Exchange Marker File Version 1.0',
```

Comment:
Here you removed an existing valid tag, namely `'Brain Vision Data Exchange Marker File Version 1.0'` - if you add this back the test should pass.

Reply:
Dear Clemens and Joan, thank you, I'm using MNE only for a couple of months so I'm not too experienced yet in how to go about this.

I also think a regex could actually be a bit better - or maybe just a header.endswith('1.0') or .endswith('2.0') to determine which version header is used. I haven't done many pull requests, so I'm now not sure what I should at this time and if my mistake was resolved by Joan already. Should I generate a new pull request, work in this request, or rather not do anything at this point? Sorry for the newbie questions.